src/server/internalServer.cpp

```diff
@@ -805,7 +814,14 @@ std::unique_ptr<Response> InternalServer::handle_content(const RequestContext& r
     if (m_verbose.load())
       printf("Failed to find %s\n", urlStr.c_str());
 
-    return Response::build_404(*this, request, bookName);
+    std::string searchURL = m_root+"/search?content="+bookName+"&pattern="+pattern;
+    MustacheData data;
+    data.set("pattern", pattern);
+    data.set("searchURL", searchURL);
+    kainjow::mustache::mustache tmpl("Make a full text search for <a href=\"{{searchURL}}\">{{{pattern}}}</a>");
```

Comment:
This template must be shared with the one above (remove the literal duplication).